lib/rubocop/cop/rspec/described_class.rb

```diff
@@ -152,6 +147,7 @@ def offensive?(node)
 
         def offensive_described_class?(node)
           return unless node.const_type?
+
```

Comment:
Styling. I found it hard to understand what guard statement this comment relates to.